### DISCUSSION TITLE
"Client error: `POST https://api.dropboxapi.com/2/files/list_folder` resulted in a `401 Unauthorized

### DISCUSSION THREAD

**Discussion by developcreativo** [spatie/flysystem-dropbox#64] [Q&A]
![image](https://user-images.githubusercontent.com/5873347/111068068-87529400-849d-11eb-9d7f-2410a2756d4b.png)
```
exception: "GuzzleHttp\Exception\ClientException"
file: "C:\laragon\www\moravaim\vendor\guzzlehttp\guzzle\src\Exception\RequestException.php"
line: 113
message: "Client error: `POST https://api.dropboxapi.com/2/files/list_folder` resulted in a `401 Unauthorized` response:↵{"error_summary": "missing_scope/", "error": {".tag": "missing_scope", "required_scope": "files.metadata.read"}}↵"
trace: [{file: "C:\laragon\www\moravaim\vendor\guzzlehttp\guzzle\src\Middleware.php", line: 69,…},…]
```

**Comment by samuelhgf** [comment]
@vdjkelly Probably you are using the DropboxClient passing oauth token. Try using authentication token instead. And set to never expire in your Dropbox app

**Comment by ligne13** [comment]
Hi,

did you find a solution ? I have the same issue.

Thanks.

**Comment by MillerAdulu** [comment]
Head over to the `Permissions` tab on your project, then enable the permissions and update them. Ensure to press submit. Then reload the page. Go back to the `Settings` tab and regenerate another token. Use the new one as the previous one doesn't get affected with the changes.
